source/agent.js

```diff
@@ -178,8 +195,14 @@ class Agent extends EventEmitter {
 						this._processQueue(name);
 					});
 
+					session.on('origin', originSet => {
+						applyOrigins();
+						// PlayerTwo.every(val => PlayerOne.includes(val));
```

Comment:
Copied from some StackOverflow post :P Good to know, we can check some condition on all items just at once.